Nikkhah/crawl_with_postid.py

```diff
@@ -0,0 +1,58 @@
+
+from instagramy import InstagramUser
+import requests
+import time
+
+
+path = r"C:/Users/PicoNet/Desktop/insta/posts/{}"
+
+
+class Crawler():
+
+    def __init__(self, file, output, session_id):
+        self.file = file
+        self.output = output
+        self.session_id = session_id
+
+    def run(self):
+        with open(self.file, "r") as file:

```

Comment:
Please seperate file open from rest of the codes because It will open until the end